src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -900,4 +900,16 @@ Please see https://aka.ms/troubleshoot_nuget_cache for more help.</value>
   <data name="Accessibility_Packages" xml:space="preserve">
     <value>Packages</value>
   </data>
+  <data name="Status_Canceled" xml:space="preserve">
```

Comment:
Did you review all the new strings with anyone yet?

Reply:
No. Those are new strings. I'm also using some existing strings in LoadingStatusIndicator

Reply:
Pick the spelling cancelled or canceled. I don't care which as long as it's consistent.

Reply:
Picked 'Canceled' as we're using american english for the english locale.